src/controllers/player.coffee

```diff
@@ -17,14 +17,25 @@ angular.module 'matching'
 	$scope.totalItems = 0
 	$scope.setCreated = false
 	$scope.helpVisible = false
+	$scope.completePerPage = []
 
 	$scope.unfinishedPagesBefore = false
 	$scope.unfinishedPagesAfter = false
+	helpModal = document.getElementById("instructions");
+	helpModal.inert = true;
```

Comment:
These lines were not causing any sort of console of behavioral failure, so they got overlooked when I reviewed your changes previously, but FYI this is not how you assign attributes to DOM elements in JS. Even though `helpModal` is referencing a DOM element object, attributes must be set (or removed) via the `setAttribute` or `removeAttribute` methods, like so:

```
helpModal.setAttribute('inert', '')
```

Bear in mind that `inert` doesn't require a value, so the mere existence of the attribute means the `inert` behavior will be applied. Therefore, `helpModal.setAttribute('inert', 'false')` wouldn't work to disable it. The attribute has to be removed completely with `removeAttribute`. This change will need to be applied on all lines that are trying to set or remove the `inert` attribute.

Reply:
This line is still improperly trying to set the `inert` property on the DOM object directly, instead of using `setAttribute()`. It's causing the test suite to fail. Always remember to run `yarn test` before committing!